app/lib/actions/pulp3/repository/create_remote.rb

```diff
@@ -3,12 +3,7 @@ module Pulp3
     module Repository
       class CreateRemote < Pulp3::Abstract
         def plan(repository, smart_proxy)
-          plan_self(:repository_id => repository.id, :smart_proxy_id => smart_proxy.id) if repository.root.url?
-        end
-
-        def run
-          repo = ::Katello::Repository.find(input[:repository_id])
-          output[:response] = repo.backend_service(smart_proxy).create_remote
+          repository.backend_service(smart_proxy).create_remote
```

Comment:
i think we need a 'if repository.root.url?'  here.  I get an error when i try to create a repo without a url.

Reply:
Updated..Had forgotten we allowed that..